app/src/androidTest/java/com/droidhats/campuscompass/views/MapFragmentTest.kt

```diff
@@ -75,4 +80,32 @@ class MapFragmentTest {
         //Ensuring the text of the toggle just clicked is indeed LOY
         onView(withId(R.id.toggleButton)).check(matches(withText("LOY")))
     }
+
+    @Test
+    fun test_additionalMenuBar(){
+        BottomSheetBehavior.from(activityRule.activity.bottom_sheet).state =
+                BottomSheetBehavior.STATE_EXPANDED
+
+        //Setting a delay to allow the bottom sheet to load
+        Thread.sleep(1000)
+
+            //Checking if building image is displayed
+            onView(withId(R.id.building_image)).check(matches(isDisplayed()))
```

Comment:
Can you please keep the indentation the same? 😄

Reply:
Yes! thank you